specifyweb/frontend/js_src/lib/components/DataModel/__tests__/businessRules.test.ts

```diff
@@ -0,0 +1,116 @@
+import { overrideAjax } from '../../../tests/ajax';
+import { mockTime, requireContext } from '../../../tests/helpers';
+import { businessRuleDefs } from '../businessRuleDefs';
+import { SerializedModel } from '../helperTypes';
+import { getResourceApiUrl } from '../resource';
+import { schema } from '../schema';
+import { Determination } from '../types';
+
+mockTime();
+requireContext();
+
+test('uniqueness rules assigned correctly', async () => {
+  const accessionAgentUniquenessRules = {

```

Comment:
This test can be simplified as `expect(businessRuleDefs).toMatchSnapshot()`

Read more about snapshot testing - https://jestjs.io/docs/snapshot-testing